docs/en/latest/getting-started.md

```diff
@@ -21,247 +21,264 @@ title: Getting Started
-        "modifiedIndex": 6,
-        "key": "/apisix/services",
-        "dir": true
-        },
-    "action": "get"
+  "node": {
+    "createdIndex": 6,
+    "modifiedIndex": 6,
+    "key": "/apisix/services",
+    "dir": true
+  },
+  "action": "get"
 }
 ```
 
-## Step 2: Create a Route in APISIX
-
-APISIX provides a powerful Admin API and a [dashboard](https://github.com/apache/apisix-dashboard) for configuring the routes/services/plugins.
-The quickstart guide will use the Admin API for configuring the routes.
+## Step 2: Create a Route
 
-A micro-service can be configured via APISIX through the relationship between several entities such as routes, services, upstream, and plugins.
-The route matches the client request and specifies how they are sent to the upstream (backend API/Service) after they reach APISIX.
-Services provide an abstraction to the upstream services. Therefore, you can create a single service and reference it in multiple routes.
-Check out the architecture document for more information.
+Congratulations! You have a running Apache APISIX instance now! Let's create a Route next!
 
-Technically all this information(upstream or service, plugins) can be included inside a route configuration. The route consists of three main parts.
+### Before we continue
 
-- Matching Rules:
+Do you know? Apache APISIX provides the powerful [Admin API](./admin-api.md) and a [Dashboard](https://github.com/apache/apisix-dashboard) for us to use, but we will use Admin API here in this guide. Let's go!
 
-    Let's take the following scenario.
-    http://example.com/services/users
+We could create one [Route](./architecture-design/route.md) and target it to our backend services (We call them [Upstream](./architecture-design/upstream.md) usually), when one `Request` reaches Apache APISIX, Apache APISIX will see where this Request should go.
 
-    The URL above hosts all the micro services related to the users(getUser/ GetAllUsers) in the system. For example the GetAllUsers endpoint can be reached via the following URL (http://example.com/services/users/GetAllUsers)
-    Now you want to expose all the `GET` endpoints(micro-services) under the `users` path. The following will be the route configuration for matching such request.
+Then how does Apache APISIX know this? that's because we have a list of rules configured with Route. Here is an sample Route data:
 
-    ```json
-    {
-        "methods": ["GET"],
-        "host": "example.com",
-        "uri": "/services/users/*",
-        ... Additional Configurations
-    }
-    ```
-
-    With the above matching rule you can communicate to APISIX via the following command.
-
-    ```bash
-    curl -i -X GET "http://{apisix_server.com}:{port}/services/users/getAllUsers?limit=10" -H "Host: example.com"
-    ```
+```json
+{
+  "methods": ["GET"],
+  "host": "example.com",
+  "uri": "/services/users/*",
+  "upstream": {
+    "type": "roundrobin",
+    "nodes": {
+      "httpbin.org:443": 1
```

Comment:
Missing `"scheme": "https"` for all the upstreams?

Reply:
ok, fine, I just replaced all 443 to 80.

1. This is a Getting Started guide, I would prefer keeping it as simple & easy to understand as possible;
2. If users want to know how to handle HTTPS cases, we could draft a guide about that particularly IMO;
3. If we use HTTPS here, I have to explain what's scheme in Upstream does...